controllers/provider-vsphere/charts/internal/shoot-system-components/charts/csi-vsphere/templates/csi-attacher-rbac.yaml

```diff
@@ -0,0 +1,77 @@
+# This YAML file contains all RBAC objects that are necessary to run external
+# CSI attacher.
+#
+# In production, each CSI driver deployment has to be customized:
+# - to avoid conflicts, use non-default namespace and different names
+#   for non-namespaced entities like the ClusterRole
+# - decide whether the deployment replicates the external CSI
+#   attacher, in which case leadership election must be enabled;
+#   this influences the RBAC setup, see below
+
+# Attacher must be able to work with PVs, nodes and VolumeAttachments
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: garden.sapcloud.io:csi-attacher
```

Comment:
Can you please rename everything to `gardener.cloud`? We should not have `garden.sapcloud.io` anywhere anymore.

Reply:
done